index.js

```diff
@@ -1,11 +1,29 @@
-function scuberGreetingForFeet(){
-  // Write your code here!
+function scuberGreetingForFeet(number){
+  let result
+  if( number < 400 ) {
+    result = 'This one is on me!';
+  } else if (number > 400 && number < 2000 ) {
+    result = 'That will be twenty bucks.';
+  } else if (number >= 2001 && number < 2500 ) {
+    result = 'I will gladly take your thirty bucks.';
+  } else if (number > 2500) {
+    result = 'No can do.';
+  }
+  return result;
 }
 
-function ternaryCheckCity(){
-  // Write your code here!
+function ternaryCheckCity(city){
+ const message = city === 'NYC' ? "Ok, sounds good." : "No go.";
+ return message;
```

Comment:
While this works, you can simply return the ternary:
```suggestion
 return city === 'NYC' ? 'Ok, sounds good.' : 'No go.;
```
Also, try to be consistent with using either single and double quotes, but not both.